### ISSUE TITLE
_pdbx_diffrn_unmerged_cell.Bravais_lattice incorrect enumeration list

### ISSUE THREAD

**Issue by CV-GPhL** [pdbxmmcifwg/diffrn-data-set-extension#12]
The list of 14 Bravais lattices is missing mP (monoclinic primitive) and includes (instead?) mI. According to Table 9.1.7.2. ("Three-dimensional Bravais lattices") in International Tables for Crystallography (2006). Vol. A, Chapter 9.1, pp. 742–749. ("9.1. Bases, lattices, Bravais lattices and other classifications", Burzlaff & Zimmermann), the list should be modified to have "mP" and no "mI".

There are other references that do include "mI" (but never dropping "mP") - so if this is needed we should decide on a reference to follow here - although the International Table Vol A one above seems to be a good starting point already.

See also: [here](https://en.wikipedia.org/wiki/Bravais_lattice) or [here](http://pd.chem.ucl.ac.uk/pdnn/symm1/trans3.htm).

**Comment by epeisach** [comment]
I agree - mP is obviously missing. If I remember from many years ago mI can be indexed as mC (mB is not listed for a similar reason). I will fix in a revision.
